BSP/Inc/BSP_Fans.h

```diff
@@ -8,6 +8,9 @@ Uses Pins PC6,7 and PB14,15
 #include "common.h"
 #include "config.h"
 
+#define DIVIDER 500 //Speed increments
+#define TOPSPEED 4000/DIVIDER //top speed achievable

```

Comment:
This seems a bit backward to me.

Reply:
I thought it would be useful for someone to know what the top speed was(when displayed in CLI). Normally if the speed is 8, someone would assume it is out of 10 and if we are in a fault state and debugging, someone who doesn't know the code would have to look through it. Also, the purpose was to make it so the inputs to the functions weren't numbers in the thousands. If it is something we should change then we can.